nh_eobs/tests/common/test_data_creator.py

```diff
@@ -5,15 +5,17 @@ def admit_and_place_patient(self):
 
 
 def admit_patient(self):
-    self.api_model = self.env['nh.eobs.api']
+    try:
+        self.api_model = self.env['nh.eobs.api']
+    except KeyError:
+        self.api_model = self.env['nh.clinical.api']
```

Comment:
Ok but this test is in nh_eobs which contains the nh.eobs.api model so there should never be an instance where nh.eobs.api isn't available for this test